Package.swift

```diff
@@ -47,9 +47,17 @@ let package = Package(
             name: "BraintreeVenmo",
             targets: ["BraintreeVenmo"]
         ),
+        .library(
+            name: "KountDataCollector",
```

Comment:
What do folks think about renaming the SPM library names for these to be something along the lines of:
- `KountDataCollector` -> `BraintreeDataCollector_BinaryDependency` or
- `PPRiskMagnes` -> `PayPalDataCollector_Dependency`      ?

Reply:
I'd lean towards leaving the names as they are, just for our own sanity. But I could be persuaded otherwise, if you think the rename would be helpful.

Reply:
I guess I was only proposing we change the name of the SPM Library that merchants will tick the box for, and keep the names for the targets, internal use (etc). 

I don't feel too strongly but wanted to throw the idea out there in case we think this might make integration less confusing for merchants.

Reply:
Right - I just meant that it might be confusing for us (or other people on the team in the future) to keep track of two sets of names.

Reply:
For what it's worth, in Android we rename the Magnes library to `paypal-data-collector` but leave Kount as `kount-data-collector`. I think the rename did make development with Magnes a little more confusing, but we continued to rename it for consistency. So, I would probably also lean towards leaving the names as they are, but maybe we should try to align the names of both across SDKs?

Reply:
Thanks for the feedback - we will keep the names as is!